.github/workflows/release.yml

```diff
@@ -40,34 +40,12 @@ jobs:
         with:
           python-version: '3.10'
 
-      # - name: Build wheels
-      #   uses: PyO3/maturin-action@v1
-      #   with:
-      #     target: ${{ matrix.target }}
-      #     container: ${{ matrix.docker_image }}
-      #     args: --release --out dist --find-interpreter --features dynamic
-      #     manylinux: auto
-      #     docker-options: |
-      #       -e RUST_BACKTRACE=1
-      #       -e DEP_JXL_LIB=${{ github.workspace }}/libjxl/build/lib
-      #       -e DEP_BROTLI_LIB=${{ github.workspace }}/libjxl/build/third_party/brotli
-      #       -e DEP_HWY_LIB=${{ github.workspace }}/libjxl/build/third_party/highway
-      #     before-script-linux: |
-      #       git clone --recurse-submodules --depth 1 -b v0.11.0 \
-      #         https://github.com/libjxl/libjxl.git
-      #       cd libjxl
-      #       cmake -B build -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF \
-      #         -DJPEGXL_ENABLE_TOOLS=OFF -DJPEGXL_ENABLE_DOXYGEN=OFF -DJPEGXL_ENABLE_MANPAGES=OFF \
-      #         -DJPEGXL_ENABLE_BENCHMARKS=OFF -DJPEGXL_ENABLE_EXAMPLES=OFF -DJPEGXL_ENABLE_JNI=OFF \
-      #         -DJPEGXL_ENABLE_SJPEG=OFF -DJPEGXL_ENABLE_OPENEXR=OFF
-      #       cmake --build build
-      #       cmake --install build
-      #       cd ..
-
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.21.2
         env:
-          CIBW_ARCHS_LINUX: auto
+          CARGO_HOME: "/usr/local/"
```

Comment:
I think you also need to set `CIBW_ENVIRONMENT_PASS_LINUX: CARGO_HOME` for this to take effect inside the container.

Reply:
Thanks! Cargo is set up now!